lib/PuppeteerSharp.Tests/FrameUtils.cs

```diff
@@ -8,14 +8,14 @@ public static class FrameUtils
     {
         public static async Task<Frame> AttachFrameAsync(Page page, string frameId, string url)
         {
-            var handle = await page.EvaluateFunctionHandleAsync(@" async (frameId, url) => {
+            var handle = (ElementHandle)await page.EvaluateFunctionHandleAsync(@" async (frameId, url) => {
```

Comment:
no null check after an `as` implied it can only be that type. hence a cast is better, since if the assumption is incorrect you will get a useful cast exception instead of a useless null ref exception